zipkin-web/src/main/resources/templates/v2/index.mustache

```diff
@@ -2,19 +2,11 @@
   <form class="form-inline" role="form">
     <div class="form-group">
       <select data-placeholder="Service Name" class="form-control input-sm" style="width: 234px;" name="serviceName" id="serviceName">
-        <option value=""></option>
```

Comment:
It's all rendered based on JSON API calls, so we should rather add cache-control headers to the API. (In zipkin-query, and zipkin-web should forward the cache headers in CopyRenderer)